sqlalchemy_jsonapi/serializer.py

```diff
@@ -939,6 +939,7 @@ def patch_resource(self, session, json_data, api_type, obj_id):
             session.rollback()
             raise ValidationError(str(e.orig))
         except AssertionError as e:
+            # pragma: no cover
```

Comment:
Added this because there is no attribute `msg` for `AssertionError` and trying to cover this leads to errors, this should be fixed in library.